.github/workflows/test-zombienet.yml

```diff
@@ -0,0 +1,49 @@
+name: Test Zombienet
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-zombienet:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      POLKADOT_VERSION: stable2407
+    steps:
+      - uses: actions/checkout@v4
+
+      - if: contains(matrix.os, 'ubuntu')
+        uses: ./.github/actions/free-disk-space
+      - if: contains(matrix.os, 'ubuntu')
+        uses: ./.github/actions/ubuntu-dependencies
+      - if: contains(matrix.os, 'macos')
+        uses: ./.github/actions/macos-dependencies
```

Comment:
Removing os if checks

```suggestion
      - uses: ./.github/actions/free-disk-space
      - uses: ./.github/actions/ubuntu-dependencies
```

Reply:
Weird, one action stopped running somehow.

![Screenshot 2024-08-15 at 18 10 17](https://github.com/user-attachments/assets/8494f181-084e-45b8-a271-62411a599643)

Tried a re-run, the same.

@Bullrich Do you have any ideas?

Reply:
Ok I know, I had [this](https://github.com/paritytech/polkadot-sdk-parachain-template/pull/17/commits/8bc0686fe22c36fc7d2f425508b6570c1735f97c) inside the action - removing it now.